examples/left_and_inner_join.cpp

```diff
@@ -60,6 +60,7 @@ inline auto initStorage(const std::string& path) {
 int main(int, char**) {
 
     auto storage = initStorage("chinook.db");
+    storage.sync_schema();
```

Comment:
no no no, here you have to download `chinook.db` instead of syncing it from here https://www.sqlitetutorial.net/sqlite-sample-database/ (text search `Download SQLite sample database`)
- [ ] remove `sync_schema`  call

Reply:
Oh oh. I see the file comment now! I was simply running through all examples... Thx